docs/frameworks/laravel.md

```diff
@@ -27,11 +27,21 @@ provider:
     runtime: provided
     environment:
         # Laravel environment variables
-        APP_STORAGE: '/tmp'
+        APP_STORAGE: /tmp
+        LOG_CHANNEL: stderr
+        SESSION_DRIVER: array
+        VIEW_COMPILED_PATH: /tmp/storage/framework/views
```

Comment:
These new variables are already defined in the `.env` file (explained below in the article). Is that necessary to add them here?

Reply:
I would argue it is. Some people just go to example code and copy/paste from it. Why not have the ideal solution in the sample app?

Reply:
The problem I see is that we define the same variables twice. That's unnecessary: it could be simpler. And having 2 times the values means that you need to modify both. Which one is actually used when deployed in production?

Reply:
ok. I simplified it now.